flybywiresim/a32nx
# [OPTIONS] Initialize unit conversion only start up

## Summary of Changes

- first #2762 needs to add two missing ecam pages

When changing the aircrafts weight unit from KG to LBS or vice versa, these changes now only become active after reloading the aircraft.
When changing the weight unit a message in amber will appear that informs the pilot to reload the aircraft.
This is done to reduce redundant checks and comparisons because the unit system of the aircraft most likely is not changed very frequent, so we figured it's smart to just initialize the unit system once on loading the aircraft.

## Screenshots (if necessary)


![image](https://user-images.githubusercontent.com/1857322/103416401-e71bed00-4b86-11eb-983e-01526e4dea05.png)


## References





## Additional context



Discord username (if different from GitHub): DerL30N#3751

## Testing instructions

1. Change the aircraft weight unit via the mcdu menu.
2. Check Upper ECAM and ECAM Pages, check mcdu readouts and entries (e.g. on fuel pred page)
   Units should not have changed.
3. Reload Aircraft
4. Repeat step 2.
   Units should now have changed.

5. Repeat steps 1 to 4 as often as you want.



## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
